src/constants/routes.ts

```diff
@@ -53,10 +53,16 @@ const paths = {
   user: "/user",
   version: "/version",
   commits: "/commits",
+  waterfall: "/waterfall",
   variantHistory: "/variant-history",
   taskHistory: "/task-history",
   jobLogs: "/job-logs",
 };
+export const redirectRoutes = {
+  projectSettingsRedirect: paths.projects,
+  userPatchesRedirect: `${paths.user}/:id`,
+  waterfallRedirect: `${paths.waterfall}/:projectIdentifier`,
+};
```

Comment:
Just pointing out something that might have gone unnoticed, but since these are now under `redirectRoutes` you could remove the trailing `"Redirect"` for each of the fields (e.g. `redirectRoutes.userPatchesRedirect` => `redirectRoutes.userPatches`). Ultimately it's up to your personal preference! 🧑‍🍳